.circleci/config.yml

```diff
@@ -220,19 +263,23 @@ workflows:
           builder_image_tag: "heroku/buildpacks:18"
           builder_file_name: buildpacks-18.tar
           requires:
-            - create-service-builder
+            - create-service-builder-18
       - publish-image:
           name: publish-18-build-stack
           image_file: pack-18-build.tar
           image_tag: heroku/pack:18-build
           image_tag_alias: heroku/pack:18-build
           requires:
-            - test-go
-            - test-java
-            - test-node-js
-            - test-ruby
-            - test-php
-            - test-python
+            - test-go-18
+            - test-go-20
```

Comment:
I think these `-20` variants need to be added as dependencies of the `publish-20-{run,build}-stack` steps, rather than the `publish-18-{run,build}-stack` steps?

Reply:
fixed